unit/models/statement.py

```diff
@@ -16,7 +16,7 @@ def from_json_api(_id, _type, attributes, relationships):
 
 OutputType = Literal["html", "pdf"]
 
-class GetStatementParams(object):
+class GetStatementParams(UnitRequest):
```

Comment:
you are not wrong, but in this case UnitRequest doesn't help too much. your decision